src/org/rascalmpl/values/RascalFunctionValueFactory.java

```diff
@@ -184,7 +184,7 @@ public org.rascalmpl.interpreter.result.Result<IValue> call(Type[] argTypes, IVa
                     throw RuntimeExceptionFactory.callFailed(ctx.getCurrentAST().getLocation(), Arrays.stream(argValues).collect(vf.listWriter()));
                 }
 
-                if (staticFunctionType.isBottom()) {
+                if (staticFunctionType.getReturnType().isBottom()) {
```

Comment:
this seem like a strange fix in the middle of a PR about the documentation?

Reply:
Well spotted! This is a nasty bug with closures generated in Java code that would not allow `void` functions at all. I have several void closures in the REPL code so this had to be fixed quickly. It's a nice separate commit though..